### DISCUSSION TITLE
Proxmox-Updater v3.8.x

### DISCUSSION THREAD

**Discussion by BassT23** [BassT23/Proxmox#59] [Announcements]
## Extras:
- Speed up start and installation

**Comment by floschad** [comment]
I got falure:

--- Searching for extra updates ---
Error during Update --- Exit Code: 129

in the LXC is only nginx installed.

**Comment by BassT23** [comment]
> I got falure:
> 
> --- Searching for extra updates --- Error during Update --- Exit Code: 129
> 
> in the LXC is only nginx installed.

-> https://github.com/BassT23/Proxmox/issues/79
